ci/helm-chart/README.md

```diff
@@ -65,6 +65,7 @@ and their default values.
 | extraArgs | list | `[]` |  |
 | extraConfigmapMounts | list | `[]` |  |
 | extraContainers | string | `""` |  |
+| extraInitContainers | string | `""` |  |
```

Comment:
Should this type be string? All other values with plural names are type list.

Reply:
This is supposed to be a string, because it contains a template, which would be resolved and is supposed to contain yaml describing list of containers

Reply:
Are you able to provide an example as a comment in the values.yank? Also please define this in the values.yaml as a null value string for readability sake. Sometimes people just look at the values and not the README so it might not be clear what values are available if not documented in values.yaml

Reply:
I kinda doubt that example would be readable. Since the value is supposed to be a template of a yaml, it would not look particularly good in an example inside of another yaml... I can make example in the README though.

Reply:
I agree, this is a fairly advanced feature the majority of users don't have to extend themselves. A short example in the readme will suffice ☺️

Reply:
Added example to [readme](https://github.com/cdr/code-server/pull/3393/files#diff-602747651030aa31de8873535c3f3757416bb752591bb8acd0401650e0891131R120)